translation/en.json

```
@@ -0,0 +1,49 @@
+{
+    "number_topic":"Topic number",
+    "comment_posted":"Comment Posted ✅",
+    "first_link_your_account":"⛔ You need to link your account first",
+    "first_link_your_account_matrix":"⛔ You need to link your account, by sending 'discourse' or '6'",
+    "writer":"🚻 Author",
+    "date":"date",
+    "publication_time":"publish date",
+    "err_active_in_the_chat":"🔴 the Bot is already active in this chat",
+    "active_bot":"Bot activated ✅",
+    "admin_activate":"⛔ You need to be an Admin to activate this Bot",
+    "category_id":"Send the Category's 🆔 you want to subscribe to",
```

`Category's` -> `category's`
```suggestion
    "category_id":"Send the category's 🆔 you want to subscribe to",
```